packages/dashboard/src/pages/Settings/regexp.ts

```diff
@@ -0,0 +1,5 @@
+export const passwordRegexp = /^(?=.{8,20}$)(?=.*[A-Z])(?=.*[a-z])(?=.*[0-9])(?=.*[^A-Za-z0-9]).*/
+
+export const emailRegexp = /^[\w-\.]+@([\w-]+\.)+[\w-]{2,4}$/
+
+export const phoneRegexp = /(\+?( |-|\.)?\d{1,2}( |-|\.)?)?(\(?\d{3}\)?|\d{3})( |-|\.)?(\d{3}( |-|\.)?\d{4})/
```

Comment:
Also, don't use regexp to match phone numbers. Do you sure your phone number can cover different phone numbers from different countries? If you really need it, please use `<input type="tel" />` or https://github.com/google/libphonenumber